README.md

```diff
@@ -62,3 +62,40 @@ The metrics exposed by this Exporter are the following.
 | `opensearch_dashboards_resp_time_max`          | Maximum response time in milliseconds                                                                                           | Gauge |
 | `opensearch_dashboards_req_disconnects`        | Request disconnections count                                                                                                    | Gauge |
 | `opensearch_dashboards_req_total`              | Total request count                                                                                                             | Gauge |
+
+
+## Debian Package
```

Comment:
I would recommend to build an LP mirror of this repo. It is easier to build the PPA + debs there.